sdk/metric/pipeline.go

```diff
@@ -233,16 +225,40 @@ func (i *inserter[N]) Instrument(inst view.Instrument, instUnit unit.Unit) ([]in
 			errs.append(err)
 		}
 	}
-	// TODO(#3224): handle when no views match. Default should be reader
-	// aggregation returned.
+
+	if !matched { // Apply implicit default view if no explicit matched.
+		a, err := i.aggregator(inst)
+		if err != nil {
+			errs.append(err)
+		}
+		if a != nil {
+			aggs = append(aggs, a)
```

Comment:
Not sure if this matter much, but should we only add `a` to the `aggs` slice if it was successfully added to the pipeline?

Same applies to the explicit views matched.

Reply:
True. I was tracking this as a secondary concern in https://github.com/open-telemetry/opentelemetry-go/issues/3240. I planned to have a follow on PR to https://github.com/open-telemetry/opentelemetry-go/pull/3238 that addressed this.

I'll update here as well and resolve the conflicts in the other.

Reply:
Updated.